src/server/scripts/Northrend/zone_grizzly_hills.cpp

```diff
@@ -854,6 +855,247 @@ class spell_infected_worgen_bite : public SpellScriptLoader
         }
 };
 
+/*######
+## Quest: Riding the Red Rocket
+######*/
+
+enum RedRocket
+{
+    SPELL_VEHICLE_WARHEAD_FUSE            = 49107,
+    SPELL_ALLIANCE_KILL_CREDIT_TORPEDO    = 49510,
+    SPELL_HORDE_KILL_CREDIT_TORPEDO       = 49340,
+    NPC_HORDE_LUMBERBOAT                  = 27702,
+    NPC_ALLIANCE_LUMBERBOAT               = 27688,
+    SPELL_DETONATE                        = 49250
+};
+
+class npc_rocket_propelled_warhead : public CreatureScript
+{
+public:
+    npc_rocket_propelled_warhead() : CreatureScript("npc_red_rocket") { }
+
+    struct npc_rocket_propelled_warheadAI : public VehicleAI
+    {
+        npc_rocket_propelled_warheadAI(Creature* creature) : VehicleAI(creature)
+        {
+            _finished = false;
+        }
+
+        uint32 _faction;
```

Comment:
All fields must be initialized in the constructor otherwise they might be accessed with random values